reference/shipping.v2.yml

```diff
@@ -1904,6 +1904,32 @@ paths:
         
         Royal Mail has no connection object properties.
         
+        ### Shipping Provider API
+        
+        Please note that this endpoint is not intended for adding connection settings. 
+        
+        Its purpose is solely to establish a connection between your BigCommerce store and the carrier. If the carrier does not have any connection settings configured, please leave the connection property empty. However, if the carrier already has connection settings configured, you can utilize this endpoint to establish the connection using the existing connection properties.
+        
+        PUT or POST example request when your carrier is not configured with any connection settings:
+        
+        ```json
+        {
+          "carrier_id" : "carrier_{your_carrier_id}",
+          "connection" : {}
+        }
+        ```
+        
+        PUT or POST example request when your carrier is configured with connection settings:
```

Comment:
```suggestion
        PUT or POST example request when you configure your carrier with connection settings:
```

Reply:
I'll leave this out because the user is not the carrier